### PR TITLE
Adding missing webhooks

### PR DESCRIPTION
**Pull request by rasyed-hw-dev**
DTPAYWONE-424

Adding missing webhooks type

- PAYPAL_ACCOUNT
- VENMO_ACCOUNT
- BUSINESS_STAKEHOLDER
- BANK_ACCOUNTS
- BANK_CARD
- PREPAID_CARD
- PAYMENTS
- TRANSFERS

### CODE REVIEW

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletWebhookNotification.java [hunk 1]
```diff
@@ -44,9 +44,81 @@
...
+        BANK_CARD_STATUS_DE_ACTIVATED("USERS.BANK_CARDS.UPDATED.STATUS.DE_ACTIVATED"),
+        BANK_ACCOUNT_DDA_CREATED("USERS.BANK_ACCOUNTS.DIRECT_DEBIT_AUTHORIZATIONS.CREATED"),
+        BANK_ACCOUNT_DDA_STATUS_ACTIVATED("USERS.BANK_ACCOUNTS.DIRECT_DEBIT_AUTHORIZATIONS.UPDATED.STATUS.ACTIVATED"),
+        BANK_ACCOUNT_DDA_STATUS_FAILED("USERS.BANK_ACCOUNTS.DIRECT_DEBIT_AUTHORIZATIONS.UPDATED.STATUS.FAILED"),
+        BANK_ACCOUNT_DDA_STATUS_CANCELLED("USERS.BANK_ACCOUNTS.DIRECT_DEBIT_AUTHORIZATIONS.UPDATED.STATUS.CANCELLED"),
+        BANK_ACCOUNT_DDA_STATUS_EXPIRED("USERS.BANK_ACCOUNTS.DIRECT_DEBIT_AUTHORIZATIONS.UPDATED.STATUS.EXPIRED"),
+        PREPAID_CARD_UPDATED_VERIFICATION_STATUS_REQUIRED("USERS.PREPAID_CARDS.UPDATED.VERIFICATION_STATUS.REQUIRED"),
+        PREPAID_CARD_UPDATED_VERIFICATION_STATUS_UNDER_REVIEW("USERS.PREPAID_CARDS.UPDATED.VERIFICATION_STATUS.UNDER_REVIEW"),
+        PREPAID_CARD_UPDATED_VERIFICATION_STATUS_VERIFIED("USERS.PREPAID_CARDS.UPDATED.VERIFICATION_STATUS.VERIFIED"),
+        PAYMENTS_CREATED("PAYMENTS.CREATED"),
```
Comment (rhunterduvar-hw-dev) [inline src/main/java/com/hyperwallet/clientsdk/model/HyperwalletWebhookNotification.java:91]: Duplicate of PAYMENT_CREATED above?

**Review by rhunterduvar-hw-dev — COMMENTED** [review]
Looks good other than previous comment, which I think needs to be fixed.

File: src/main/java/com/hyperwallet/clientsdk/model/HyperwalletWebhookNotification.java (hunk 1 above)
Comment (rasyed-hw-dev) [reply src/main/java/com/hyperwallet/clientsdk/model/HyperwalletWebhookNotification.java:91]: removed it

**Review by rhunterduvar-hw-dev — APPROVED** [review]

**Review by rhunterduvar-hw-dev — APPROVED** [review]

File: src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java [hunk 2]
```diff
@@ -6494,6 +6495,27 @@ public void testListWebhookEvents_withParameters() throws Exception {
...
+                {"USERS.BUSINESS_STAKEHOLDERS.UPDATED.VERIFICATION_STATUS.REQUIRED", HyperwalletWebhookNotification.Type.USER_BUSINESS_STAKEHOLDER_UPDATED_VERIFICATION_STATUS_REQUIRED},
+                {"USERS.BANK_CARDS.CREATED", HyperwalletWebhookNotification.Type.BANK_CARD_CREATED}, {"USERS.BANK_ACCOUNTS.DIRECT_DEBIT_AUTHORIZATIONS.CREATED", HyperwalletWebhookNotification.Type.BANK_ACCOUNT_DDA_CREATED},
+                {"PAYMENTS.CREATED", HyperwalletWebhookNotification.Type.PAYMENTS_CREATED}, {"TRANSFERS.UPDATED.STATUS.SCHEDULED", HyperwalletWebhookNotification.Type.TRANSFERS_UPDATED_STATUS_SCHEDULED},
+                {"TRANSFERS.REFUND.CREATED", HyperwalletWebhookNotification.Type.TRANSFERS_REFUND_CREATED}, {"PAYMENT_ROUTING_ACCOUNTS.CREATED", HyperwalletWebhookNotification.Type.PAYMENT_ROUTING_ACCOUNTS_CREATED}};
+    }
+
+    @Test(dataProvider = "webhook")
+    public void parameterTest(String myName, Object abc) {
+        System.out.println("Parameterized value is : " + myName + " " + abc.toString());
+        Assert.assertNotNull(abc.toString());
```
Comment (asingh-hw) [inline src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java:6515]: Before Assert, get the Type from the Map. i.e. 
```
Type type = HyperwalletWebhookNotification.Type.getType(myName);
Assert.assertNotNull(type);
Assert.assertEquals(type.toString(), abc.toString());
```

Comment (rasyed-hw-dev) [reply src/test/java/com/hyperwallet/clientsdk/HyperwalletTest.java]: Done

**Review by asingh-hw — APPROVED** [review]

**Pull request merged**